packages/backend/src/execution/middleware.test.ts

```diff
@@ -54,8 +61,31 @@ describe('createExpressSynthqlHandler', async () => {
         const result = await queryEngine.executeAndWait(q, { context });
 
         const resultFromQueryWithContextManuallyAdded =
-            await queryEngine.executeAndWait(queryWithContextManuallyAdded);
+            await queryEngine.executeAndWait(queryWithContextManuallyAdded, {
+                context: { permissions: context.permissions },
+            });
 
         expect(result).toEqual(resultFromQueryWithContextManuallyAdded);
     });
 });
+
+function createPaymentQuery() {
+    return from('payment')
+        .permissions('user:read')
+        .include({
+            customer: from('customer')
+                .permissions('admin:read')
+                .where({
+                    customer_id: col('payment.customer_id'),
+                })
+                .one(),
+        })
+        .groupBy(
```

Comment:
why the huge groupBy?

Reply:
I'm not actually sure, but not including all of them is causing it to throw a aggregate error